pom.xml

```diff
@@ -108,6 +109,13 @@
 				<version>${project.version}</version>
 			</dependency>
 
+			<dependency>
+				<groupId>org.glassfish.jersey.core</groupId>
+				<artifactId>jersey-common</artifactId>
+				<version>${jersey.common.version}</version>
```

Comment:
It was added because of Response class for the JUnit tests, I don't know why JAX RS API was not enough for it.